lib/active_model/serializer/adapter.rb

```diff
@@ -94,6 +98,38 @@ def include_meta(json)
         json[meta_key] = meta if meta
         json
       end
+
+      def key_format
+        serializer.key_format || default_key_format
```

Comment:
So, this essentially allows a serializer to 'override' the default_key_format for the current adapter?